test/CMakeLists.txt

```diff
@@ -25,6 +25,12 @@ target_link_libraries( quadrature_manipulation PUBLIC Catch2::Catch2WithMain int
 add_executable( 1d_quadratures 1d_quadratures.cxx )
 target_link_libraries( 1d_quadratures PUBLIC Catch2::Catch2WithMain integratorxx )
 
+add_executable( gausslegendre gausslegendre.cxx )
+target_link_libraries( gausslegendre PUBLIC Catch2::Catch2WithMain integratorxx )
+
+#add_executable( gausslobatto gausslobatto.cxx )
+#target_link_libraries( gausslobatto PUBLIC Catch2::Catch2WithMain integratorxx )
```

Comment:
Why is this commented out?

Also, if these are to be run in the unit test system, you must also add appropriate `add_test` lines to register them with `ctest`

Reply:
Like I commented above, these are commented out since Lobatto quadrature isn't yet in the master branch!

Reply:
Tests added